lib/src/models/bar/chart_bar_data_item.dart

```diff
@@ -1,9 +1,16 @@
 part of 'chart_bar_layer.dart';
 
+/// Provides data item of bar.
```

Comment:
Maybe more details here? 
Something like `A collection of values for a single bar in a chart`